src/Tools/dotnet-monitor/InProcessFeatures/InProcessFeatures.cs

```diff
@@ -32,10 +32,8 @@ internal sealed class InProcessFeatures : IInProcessFeatures
 
         public bool IsMutatingProfilerRequired => IsParameterCapturingEnabled;
```

Comment:
Now that parameter capturing lives entirely in the startup hook, it **should** work on attach. In theory all of the code we have in the mutating profiler should support this but we've never validated it. Have you had been able to validate parameter capturing works when connecting to an already running process?  (We might want to add a test case for this seperately -- we don't need to exercise the whole parameter capturing path (unless its easy), just that function probes work

Reply:
If it does work, I think we can adapt the existing `ParameterCapturingTests` to test both connect and listen mode.

Reply:
I mean validating that this working in `Listen` + no suspend mode, we don't load in-proc features in Connect